src/viewer/core/ViewerWindowManager.C

```diff
@@ -1338,6 +1341,36 @@ ViewerWindowManager::MoveAndResizeWindow(int windowIndex, int x, int y,
 void
 ViewerWindowManager::ResizeWindow(int windowIndex, int w, int h)
 {
+#if defined(__APPLE__)
```

Comment:
This logic has two approachs. One, currently commented out, briefly switches window size in Y dimension by a small amount. I like it the best. But, it doesn't fully work because the `origy` is NOT the true size of the window at startup.

The second approach briefly swaps layouts...ugly...but it returns the window to its true orig. size too.

This will happen only ever once upon first plot and only on macos

Reply:
This logic is triggered twice by two calls from `ViewerMethods.C::DrawPlots()`